website/examples/customization-fixedweeks.tsx

```diff
@@ -2,5 +2,5 @@ import React from "react";
 import { DayPicker } from "react-day-picker";
 
 export function Example() {
-  return <DayPicker showOutsideDays fixedWeeks />;
+  return <DayPicker fixedWeeks />;
```

Comment:
I thought fixedWeeks also required showOutsideDays. Did you change that?

Reply:
Good catch thanks, restored in b2ba94237a8427021dac77a4baaafd953352ddef.